### DISCUSSION TITLE
part-ii/2021-118/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#400] [Question comments]
Part II 2021 Principles of Statistics: Paper 2, Section II, J 

https://questions.tripos.org/part-ii/2021-118/

**Comment by ArthurConmy** [comment]
Seems pretty brutal to figure out how to push the symbols correctly under timed conditions.

<details>
<summary>b)</summary>

Compare $X_1 + ... + X_n$ to the $m$ different things $(Z_1 + ... + Z_n)$, $(Z_{n+1}+Z_{n+2}+...+Z_{2n})$, ... and if it's the largest then reject $H_0$.

</details> 

<details>
<summary>c)</summary>

$\mathbb{E}\left[\left(\hat{f}_{n}(x)-f(x)\right)^{2}\right] \leqslant \frac{2}{n^{2 / 3}} = \text{Var}( \hat{f}_n(x) ) - ( f - \mathbb{ E}[ \hat{f}_n(x) ] )^2 $. 

The latter term is $ \frac{1}{16n^{2/3}} $ and the former term is variance of a scaled binomial random variable, where the parameter is at most $ f(x)/n^{1/3} + 1/4n^{2/3} $ by the $|f'| \le 1$ condition. So putting this together, we need show 

$$ \frac{f(x)}{n^{2/3}} + \frac{1}{4n} + \frac{1}{16n^{2/3}} \le \frac{2}{n^{2/3}} $$

which is true.

</details>
